Code/Entities/ThrowBox.cs

```diff
@@ -434,7 +469,9 @@ private void Shatter() {
 
                 for (int i = 0; i < Width / 8f; i++) {
                     for (int j = 0; j < Height / 8f; j++) {
-                        if (_isMetal) {
+                        if (_debrisTypeOverride != '\0') {
```

Comment:
Perhaps it would make more sense to use a nullable `char` rather than checking for a magic `\0`